Modules/@babylonjs/react-native-basekit/README.md

```diff
@@ -0,0 +1,120 @@
+# Babylon React Native Basekit
+
+## Usage
+
+This quick overview will help you understand the constructs provided by Babylon React Native and how to use them in a React Native application.
+
+### Dependencies
+
+This package has several **peer dependencies**. If these dependencies are unmet, `npm install` will emit warnings. Be sure to add these dependencies to your project.
+
+### Installation Configuration
+
+Downloaded dependencies are cache by default in directory specifiec by `npm_config_cache`. It's possible to specify a new cache directory by setting `npm_config_babylon_binary_cache` environment variable.
+
+Also, it's possible to set the source of these downloaded dependencies. By default, it's using main Github repo releases: `https://github.com/BabylonJS/BabylonReactNative/releases/download`.
+Other possibilities are:
+- By command argument for `npm install` : `--brn-binary-site=https://mysite.com/downloads`
+- By environment variable `export npm_config_brn_binary_site=https://mysite.com/downloads`
+- By configuration in package.json: `nodeBabylonConfig: { binarySite: 'https://mysite.com/downloads'}`
+
+These case location must respect same hierarchy as the main Github repo releases.
+
+### Android Configuration
+
+The minimum Android SDK version is 18. This must be set as `minSdkVersion` in the consuming project's `build.gradle` file.
+
+### iOS Configuration
+
+The minimum deployment target version is 12. This must be set as `iOS Deployment Target` in the consuming project's `project.pbxproj`, and must also be set as `platform` in the consuming project's `podfile`.
+
+### Platform Native Packages
+
+Babylon React Native supports react-native from 0.69 to 0.71+. See project Github main readme for supported versions.
+
+### `useEngine`
```

Comment:
Is this duplicated documentation? Is the plan that once we are fully on this new packaging scheme, we delete the old stuff?

Reply:
yes, it's duplicated from @babylonjs/react-native readme. long term is indeed to replace old legacy packages by this one.